apps/chat/bots.py

```diff
@@ -84,13 +84,16 @@ def _initialize(self):
     def _call_predict(self, input_str, save_input_to_history=True):
         if self.child_chains:
             chain = self._get_child_chain(input_str)
+            add_experiment_tag = True
```

Comment:
Not blocking but if we made `_get_child_chain` also return the keyword then we could pass that in directly instead of the flag and we wouldn't need to look up the route in the runnable.